### PR TITLE
Changes to Sites-Available

### PR DESCRIPTION
**Pull request by teolemon**
fix-stuff

### CODE REVIEW

File: conf/nginx/sites-available/off [hunk 1]
```diff
@@ -356,11 +356,11 @@ server {
...
         if ($host = androidtest.openfoodfacts.org) { return 302 https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner ; }
-        if ($host = ios.openfoodfacts.org) { return 302 https://itunes.apple.com/app/open-food-facts/id588797948 ; }
-        if ($host = iostest.openfoodfacts.org) { return 302 https://itunes.apple.com/app/open-food-facts/id588797948 ; }
-        if ($host = slack.openfoodfacts.org) { return 302 https://slack-openfoodfacts.herokuapp.com ; }
-        if ($host = donner.openfoodfacts.org) { return 302 https://www.helloasso.com/associations/open-food-facts/collectes/aidez-open-food-facts-a-ameliorer-l-alimentation-de-tous ; }
-        if ($host = donate.openfoodfacts.org) { return 302 https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget/en ; }
+        if ($host = ios.openfoodfacts.org) { return 302 https://apps.apple.com/app/open-food-facts/id588797948 ; }
+        if ($host = iostest.openfoodfacts.org) { return 302 https://apps.apple.com/app/open-food-facts/id588797948 ; }
+        if ($host = slack.openfoodfacts.org) { return 302 https://slack.openfoodfacts.org ; }
```
Comment (stephanegigandet) [inline conf/nginx/sites-available/off]: That's the redirect from slack.openfoodfacts.org to the actual url, we need to keep it.

File: conf/nginx/sites-available/off [hunk 2]
```diff
@@ -105,9 +105,9 @@ server {
 	if ($host = translations.openfoodfacts.org) { return 302 https://translate.openfoodfacts.org/ ; }
 	if ($host = android.openfoodfacts.org) { return 302 https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner ; }
         if ($host = androidtest.openfoodfacts.org) { return 302 https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner ; }
-        if ($host = ios.openfoodfacts.org) { return 302 https://itunes.apple.com/app/open-food-facts/id588797948 ; }
-        if ($host = iostest.openfoodfacts.org) { return 302 https://itunes.apple.com/app/open-food-facts/id588797948 ; }
-        if ($host = slack.openfoodfacts.org) { return 302 https://slack-openfoodfacts.herokuapp.com ; }
+        if ($host = ios.openfoodfacts.org) { return 302 https://apps.apple.com/app/open-food-facts/id588797948 ; }
+        if ($host = iostest.openfoodfacts.org) { return 302 https://apps.apple.com/app/open-food-facts/id588797948 ; }
+        if ($host = slack.openfoodfacts.org) { return 302 https://slack.openfoodfacts.org ; }
```
Comment (VaiTon) [inline conf/nginx/sites-available/off]: See https://github.com/openfoodfacts/openfoodfacts-server/pull/2736/files#r355177833

File: conf/nginx/sites-available/off [hunk 3]
```diff
@@ -536,11 +536,11 @@ server {
...
         if ($host = androidtest.openfoodfacts.org) { return 302 https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner ; }
-        if ($host = ios.openfoodfacts.org) { return 302 https://itunes.apple.com/app/open-food-facts/id588797948 ; }
-        if ($host = iostest.openfoodfacts.org) { return 302 https://itunes.apple.com/app/open-food-facts/id588797948 ; }
-        if ($host = slack.openfoodfacts.org) { return 302 https://slack-openfoodfacts.herokuapp.com ; }
-        if ($host = donner.openfoodfacts.org) { return 302 https://www.helloasso.com/associations/open-food-facts/collectes/aidez-open-food-facts-a-ameliorer-l-alimentation-de-tous ; }
-        if ($host = donate.openfoodfacts.org) { return 302 https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget/en ; }
+        if ($host = ios.openfoodfacts.org) { return 302 https://apps.apple.com/app/open-food-facts/id588797948 ; }
+        if ($host = iostest.openfoodfacts.org) { return 302 https://apps.apple.com/app/open-food-facts/id588797948 ; }
+        if ($host = slack.openfoodfacts.org) { return 302 https://slack.openfoodfacts.org ; }
```
Comment (VaiTon) [inline conf/nginx/sites-available/off]: See https://github.com/openfoodfacts/openfoodfacts-server/pull/2736/files#r355177833

**Comment by VaiTon** [comment]
I changed a lot of whitespaces. To see the changed files without whitespaces changes follow [this link](https://github.com/openfoodfacts/openfoodfacts-server/pull/2736/files?w=1).

**Pull request merged**